src/components/FixedSideContainer.tsx

```diff
@@ -14,7 +15,11 @@ export const FixedSideContainer = ({
   className,
 }: FixedSideContainerProps) => (
   <div
-    className={`FixedSideContainer FixedSideContainer_side_${side} ${className}`}
+    className={clsx(
+      `FixedSideContainer`,
+      { [`FixedSideContainer_side_${side}`]: side },
```

Comment:
unnecessary as `side` cannot be falsy, and this makes it less readable, so I'd keep it inside the template string

Reply:
Fixed